utils/onboarding/weblog_interface.py

```diff
@@ -5,9 +5,15 @@
 
 
 def make_get_request(app_url):
-    generated_uuid = str(randint(0, 100000000000000000))
+    generated_uuid = str(randint(1, 100000000000000000))
     requests.get(
-        app_url, headers={"x-datadog-trace-id": generated_uuid, "x-datadog-parent-id": generated_uuid}, timeout=10
+        app_url,
+        headers={
+            "x-datadog-trace-id": generated_uuid,
+            "x-datadog-parent-id": generated_uuid,
+            "x-datadog-sampling-priority": "2",
```

Comment:
not having the priority set would make sampling trigger. Since auto-instrument generates multiple traces before the rate applied might not be of 1.
priority =2 enforces that it's sent to backend!